beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -205,7 +198,7 @@ func (s *Service) validateCommitteeIndexBeaconAttestation(ctx context.Context, p
 		}()
 	}
 
-	s.setSeenCommitteeIndicesSlot(data.Slot, data.CommitteeIndex, att.GetAggregationBits())
+	s.setSeenCommitteeIndicesSlot(data.Slot, committeeIndex, att.GetAggregationBits())
```

Comment:
we should double check what other parts of the code uses data.committeeIndex instead of getting the appropriate committee index based on fork version